gradlew

```diff
@@ -114,6 +109,7 @@ fi
 if $cygwin ; then
     APP_HOME=`cygpath --path --mixed "$APP_HOME"`
     CLASSPATH=`cygpath --path --mixed "$CLASSPATH"`
+    JAVACMD=`cygpath --unix "$JAVACMD"`
```

Comment:
Context for the change in gradle can be seen in [GRADLE-2673](https://issues.gradle.org/browse/GRADLE-2673)